code/modules/SCP/SCPs/SCP-513.dm

```diff
@@ -15,9 +15,6 @@
 	START_PROCESSING(SSobj, src)
 
 /obj/item/scp513/proc/ring(mob/living/user)
-	if(user in victims)
-		to_chat(user, "<span class='notice'>I rung it once, and I felt terrible. Why the hell would I that again?!</span>")
-		return
```

Comment:
Why dod you remove these?

Reply:
I moved it to attack_hand, so the message won't show if someone who has already rung it picks it up on harm intent